launcher/minecraft/MinecraftUpdate.cpp

```diff
@@ -16,32 +16,22 @@
 #include "MinecraftUpdate.h"
 #include "MinecraftInstance.h"
 
-#include <QDataStream>
-#include <QFile>
-#include <QFileInfo>
-#include <QTextStream>
-
-#include <FileSystem.h>
-#include "BaseInstance.h"
-#include "minecraft/Library.h"
 #include "minecraft/PackProfile.h"
 
+#include "tasks/SequentialTask.h"
 #include "update/AssetUpdateTask.h"
 #include "update/FMLLibrariesTask.h"
 #include "update/FoldersTask.h"
 #include "update/LibrariesTask.h"
 
-#include <meta/Index.h>
-#include <meta/Version.h>
-
-MinecraftUpdate::MinecraftUpdate(MinecraftInstance* inst, QObject* parent) : Task(parent), m_inst(inst) {}
+MinecraftUpdate::MinecraftUpdate(MinecraftInstance* inst, QObject* parent) : SequentialTask(parent), m_inst(inst) {}
 
 void MinecraftUpdate::executeTask()
 {
-    m_tasks.clear();
+    m_queue.clear();
```

Comment:
No other subclass of SequentialTask seems to do this. Any reason for it here?

Reply:
Yes because I populate the m_queue when the task is executed.
So in case I retry it I need the queue to be repopulated again.